### ISSUE TITLE
Python vs. JavaScript: A Comparison of Two Beginner-Friendly Languages

### ISSUE THREAD

**Issue by VIBER1** [Grimm-s-Alchemy-Chamber/hackers_hive#91]
# Title Of Blog Post : Python vs. JavaScript: A Comparison of Two Beginner-Friendly Languages

**Comment by swaleha2002** [comment]
Hi there!

I've reviewed the issue regarding Python vs. JavaScript: A Comparison of Two Beginner-Friendly Languages. I'd be happy to work on this.

Could you please assign this issue to me? I'll make the necessary updates promptly.

Thank you!

**Comment by VIBER1** [comment]
yeah sure you can work on it and please refer to contributing guidelines.

**Comment by swaleha2002** [comment]
sure, thank you @VIBER1

**Comment by AnuragDhavale** [comment]
@VIBER1 I would like to work on it, please assign me
